experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -93,8 +91,7 @@ class ExPlat internal constructor(
     }
 
     private fun getAssignments(refreshStrategy: RefreshStrategy): Assignments {
-        val cachedAssignments: Assignments =
-            runBlocking { repository.getCached() } ?: Assignments(emptyMap(), 0, 0)
+        val cachedAssignments: Assignments = repository.getCached() ?: Assignments(emptyMap(), 0, -1)
```

Comment:
Question (❓): What's with the `-1` change? 🤔

Reply:
TLDR; I replaced this with `getAssignments` possible to return `null` now. I think it's much more readable now, and makes more sense: if there's no cache, it's better to return `null` than an `Assignment` with made up values. 98bba0a58977c64dac88429d5e1f07c3ad0d380a

Context here was that I wanted this Assignment to be `stale` when `Clock` is `0` (in tests, making things more readable). It was also ok for my thinking process to make `fetchedAt` value a minus value (that doesn't make sense) to signal that it's not a valid object. But as I moved to use `null`, this doesn't matter.

Reply:
Great, thanks for this change @wzieba , I agree, this seems much more robust to me! 💯